packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -20,6 +21,33 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     this.db = db;
     this.tableMap = buildModelTableMap(baseType);
     this.collectionName = this.tableMap.tableName;
+    Object.keys(baseType.getFields()).forEach(k => {
+      const relationshipData = parseRelationshipAnnotation(baseType.getFields()[k].description)
+      if (relationshipData?.kind && ['manyToOne', 'manyToMany'].includes(relationshipData.kind)) {
+        this.db.collection(this.collectionName).createIndex({
+          [relationshipData.key]: 1
+        }, (e, r) => {
+          if (e) {
+            console.error(e);
+          } else {
+            console.log(JSON.stringify(r,null,4))
+          }
+        });
+      }
+
+      const otherAnnotations = parseAnnotations('db', baseType.getFields()[k].description) as any;
+      if (otherAnnotations?.index) {
```

Comment:
If someone changes the indexed column - there should probably also be a scan to remove old index?

I don't know a lot about Mongo, so forgive if I am way off :D

Reply:
Without support for renames changes for columns in mongo are not allowed